tests/tui/test_tui_coverage_gaps.py

```
@@ -0,0 +1,667 @@
+        panel.update = MagicMock()
+        panel.set_stats(groups=5, space_wasted="200 MB", recoverable="100 MB")
+        rendered = panel.update.call_args[0][0]
+        assert "5" in rendered
+        assert "200 MB" in rendered
+
+
+class TestAnalyticsViewMountCompose:
+    def test_compose_yields_widgets(self):
+        """compose() must yield StaticWidgets — test by verifying it's iterable."""
+        from file_organizer.tui.analytics_view import AnalyticsView
+        view = AnalyticsView(directory="/tmp")
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)